artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2073,9 +2074,16 @@
    </xsd:complexType>
 
    <xsd:complexType name="brokerConnectType">
-      <xsd:sequence maxOccurs="unbounded">
+      <xsd:sequence maxOccurs="unbounded" minOccurs="0">
         <xsd:element name="amqp-connection" type="amqp-connectionUriType"/>
       </xsd:sequence>
+      <xsd:attribute name="mirror-id" type="xsd:unsignedByte" use="required">
+         <xsd:annotation>
+            <xsd:documentation>
+               When you connect mirrors towards this broker, you need a uniqute byte (less than 255, more than 0) to identify your broker.
+            </xsd:documentation>
+         </xsd:annotation>
+      </xsd:attribute>
```

Comment:
I meant this to be optional. as a matter of fact.. I had optional at some point, and I guess I git rollback by accident a bunch of stuff around that. It will be optional.